docs/report/powerbi/sample-testplans-aggregatedatlevel.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Aggregated view for test suites sample Power BI report 
+titleSuffix: Azure DevOps
+description: Sample Power BI queries to generate aggregated view for test suites
+ms.prod: devops
+ms.technology: devops-analytics
+ms.reviewer: ravishan
+ms.manager: pkuma
+ms.author: shdalv
+ms.custom: powerbisample
+author: KathrynEE
+ms.topic: sample
+monikerRange: '>= azure-devops-2019' --TODO
```

Comment:
I request guidance for filling out this field, could you please provide more information about what monikerRange is?